pkg/controller/restore_controller_test.go

```diff
@@ -148,37 +147,45 @@ func TestProcessRestore(t *testing.T) {
 		},
 		{
 			name:        "restore with both namespace in both includedNamespaces and excludedNamespaces fails validation",
-			restore:     NewTestRestore("foo", "bar", api.RestorePhaseNew).WithBackup("backup-1").WithIncludedNamespace("another-1").WithExcludedNamespace("another-1").Restore,
+			restore:     NewRestore("foo", "bar", "backup-1", "another-1", "*", api.RestorePhaseNew).WithExcludedNamespace("another-1").Restore,
```

Comment:
Can't say I'm crazy about switching from explicit With* helpers to a big constructor with lots of args.

Reply:
My goal here was to reduce the verbosity of constructing typical test objects, given that most of them require the same set of values to be set. I thought this was easier to read.

Reply:
If you think it's easier, I won't block on this